src/main/java/com/avengers/gamera/repository/UserRepository.java

```diff
@@ -7,7 +7,8 @@
 import java.util.Optional;
 
 @Repository
-public interface UserRepository extends JpaRepository<User,Long> {
+public interface UserRepository extends JpaRepository<User, Long> {
     Boolean existsUserByEmail(String email);
```

Comment:
We normally use primitive type rather than boxed type unless the type should be nullable.

Reply:
JpaRepository doesn't accept primitive type